src/Vary/VEither.hs

```diff
@@ -148,14 +155,18 @@ veither _ g (VRight y)    =  g y
 
 -- Matches when the VEither contains one of the errors, returning @Vary errs@
 pattern VLeft :: forall a errs. Vary errs -> VEither errs a
-{-# INLINE VLeft #-}
+#if __GLASGOW_HASKELL__ >= 902
+-- {-# INLINE VLeft #-}
```

Comment:
Isn't commented out pragma basically disabled? Shouldn't this be uncommented?

Reply:
Fixed.  Thanks!